.travis.yml

```diff
@@ -1,15 +1,16 @@
+dist: xenial
 language: python
 sudo: false
 python:
-  - "3.4"
-  - "3.5"
   - "3.6"
+  - "3.7"
 cache:
   pip: true
 install:
-  - pip install tox-travis codecov
+  - pip install tox-travis codecov black==19.3b0
 script:
   - tox
+  - black --check .
```

Comment:
I would change the order of scripts. There is no need to wait for the tests if the code format is invalid.

Reply:
I think code being broken is more important than it being ugly, otherwise, someone will merge it thinking only formatting is incorrect.